utils/command_build_cmake_clib.py

```diff
@@ -109,7 +109,8 @@ def run(self):
             "-DDPNP_SYCL_QUEUE_MGR_ENABLE:BOOL=" + _dpctrl_exists,
             "-DDPNP_QUEUEMGR_INCLUDE_DIR=" + _dpctrl_include_dir,
             "-DDPNP_QUEUEMGR_LIB_DIR=" + _dpctrl_library_dir,
-            "-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON"
+            "-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON",
+            "-DENABLE_TESTS=ON"
```

Comment:
@shssf Do you want me to add gtest check for `WIN` env or switch `ENABLE_TEST` off for `WIN`?